tests/util.py

```diff
@@ -39,6 +40,39 @@ def assert_xml_equal(actual: Union[IO, str], expected: Union[IO, str]):
         raise e
 
 
+def assert_json_equal(actual: str, expected: Union[IO, str]):
+    """Checks whether the JSON trees behind `actual` and `expected` are equal.
+
+    Args:
+        actual: The actual JSON
+        expected: The expected JSON
+
+    Throws:
+        AssertionError when json(actual) != json(expected)
+    """
+    actual_json = json.dumps(json.loads(actual), sort_keys=True, indent=2)
```

Comment:
Is that really the best way to compare two jsons? I would expect dict comparison to also work

Reply:
Well, it works and it produces a nicely diffable representation. No idea if it is the best way. What benefit would dict-comparison have?

Reply:
It would be independent of the ordering. You can always add the diffable representation after the assert, e.g. `assert foo, "My error message`

Reply:
This is also independent of ordering because keys are sorted - but to be honest, I'd remove the key sorting because order actually matters. E.g. we want that the three main section have a specific order, also in an FS, the fields with the reserved names must come before the feature fields, etc.